src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -464,6 +568,68 @@ def schema_version(self) -> tuple[int, int, int, int]:
             number = re.search(prefix + r"(\d)", schema)
             version.append(int(number.group(1)) if number else 0)
         return tuple(version)
+    
+    @property 
+    def mvd(self) -> str:
+        """
+        View Definition supported by the exporting application that is reflected in the IFC file
+        For example: 
+        “CoordinationView”
```

Comment:
You have some really fancy quotes, better remove them or repalce them with the plain old ascii ones

```
>>> hex(ord('”'))
'0x201d'
```